src/SFA.DAS.ApplyService.Data/Queries/OversightReviewQueries.cs

```diff
@@ -45,7 +45,9 @@ public async Task<PendingOversightReviews> GetPendingOversightReviews(string sea
                               FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
                          LEFT JOIN OversightReview r ON r.ApplicationId = apply.ApplicationId
```

Comment:
Is it me or isn't this line doing anything? Should you also check  'AND r.ApplicationId IS NULL'